buildspec_publish_sar.yml

```diff
@@ -13,8 +13,6 @@ env:
     REACT_APP_ORCID_CLIENT_SECRET: '${REPLACE_ORCID_CLIENT_SECRET}'
     REACT_APP_ORCID_REDIRECT_URI: '${REPLACE_ORCID_REDIRECT_URI}'
     REACT_APP_ORCID_BASE_URL: '${REPLACE_ORCID_BASE_URL}'
-    REACT_APP_PUBLICATION_CHANNEL_API_URL: '${REPLACE_PUBLICATION_CHANNEL_API_URL}'
-    REACT_APP_AUTHORITY_REGISTER_API_URL: '${REPLACE_AUTHORITY_REGISTER_API_URL}'
```

Comment:
I think we should add a comment here that describes for our future selfs why we are doing this.